### PR TITLE
feat: add tx state on the top right button.

### PR DESCRIPTION
**Pull request by fricoben**
This PR permits having a transaction state on the button only on the minting of identities.

We should add it everywhere there is a tx made by the user but the problem is that Starknet React does not work properly as we have to manually `removeTransaction` which means that we'll need to follow the state of all the txs (on the identity actions components it would add 500 lines of code to follow each type of TXs). 

I prefer to wait for a working version of Starknet React to do it.

### CODE REVIEW

File: components/UI/navbar.tsx [hunk 1]
```diff
@@ -63,6 +75,24 @@ const Navbar: FunctionComponent = () => {
     }
   }, [library, network, isConnected]);
 
+  useEffect(() => {
+    if (transactions) {
+      console.log("transactions", transactions);
```
Comment (Th0rgal) [inline components/UI/navbar.tsx]: please remove the console.log

File: pages/identities.tsx [hunk 2]
```diff
@@ -35,31 +37,44 @@ const Identities: NextPage = () => {
...
+
   useEffect(() => {
     if (account) {
       // Our Indexer
       setLoading(true);
       fetch(
-        `${process.env.NEXT_PUBLIC_SERVER_LINK}/addr_to_full_ids?addr=${hexToDecimal(account.address)}`
+        `${
+          process.env.NEXT_PUBLIC_SERVER_LINK
+        }/addr_to_full_ids?addr=${hexToDecimal(account.address)}`
```
Comment (Th0rgal) [inline pages/identities.tsx:67]: IfI'm not mistaken, this should work even without hexToDecimal

File: pages/identities.tsx [hunk 3]
```diff
@@ -35,31 +37,44 @@ const Identities: NextPage = () => {
...
           setOwnedIdentities(data.full_ids);
         });
 
       fetch(
-        `${process.env.NEXT_PUBLIC_SERVER_LINK}/addr_to_external_domains?addr=${hexToDecimal(
-          account.address
-        )}`
+        `${
+          process.env.NEXT_PUBLIC_SERVER_LINK
+        }/addr_to_external_domains?addr=${hexToDecimal(account.address)}`
```
Comment (Th0rgal) [inline pages/identities.tsx:77]: here too

File: components/identities/actions/identityActions.tsx [hunk 4]
```diff
@@ -83,7 +88,11 @@ const IdentityActions: FunctionComponent<IdentityActionsProps> = ({
     if (address && tokenId) {
       setLoading(true);
       // Our Indexer
-      fetch(`${process.env.NEXT_PUBLIC_SERVER_LINK}/addr_to_full_ids?addr=${hexToDecimal(address)}`)
+      fetch(
+        `${
+          process.env.NEXT_PUBLIC_SERVER_LINK
+        }/addr_to_full_ids?addr=${hexToDecimal(address)}`
```
Comment (Th0rgal) [inline components/identities/actions/identityActions.tsx:94]: IfI'm not mistaken, this should work even without hexToDecimal

**Review by Th0rgal — APPROVED** [review]
this looks good to me, please just remove the console.log and may be the hexToDecimal (but you can keep them if you prefer) :)

File: pages/identities.tsx (hunk 2 above)
Comment (fricoben) [reply pages/identities.tsx:67]: No it does not

**Pull request merged**